src/platform/packages/shared/kbn-esql-validation-autocomplete/src/autocomplete/commands/stats/index.ts

```diff
@@ -26,24 +32,35 @@ export async function suggest(
   _columnExists: (column: string) => boolean,
   getSuggestedVariableName: () => string,
   _getExpressionType: (expression: ESQLAstItem | undefined) => SupportedDataType | 'unknown',
-  getPreferences?: () => Promise<{ histogramBarTarget: number } | undefined>
+  getPreferences?: () => Promise<{ histogramBarTarget: number } | undefined>,
+  fullTextAst?: ESQLAst,
```

Comment:
heads-up `fullTextAst` was removed in https://github.com/elastic/kibana/pull/210607

Reply:
No I am happy you changed it, I was going to do so in a follow up PR. Merged with main here https://github.com/elastic/kibana/pull/210170/commits/8989a6412cc00b254b5e9fa3701725c079c77894